LocalSettings.php

```diff
@@ -1707,8 +1707,16 @@
 	],
 
 	// Gallery Options
-	'+wgGalleryOptions' => [
-		'default' => [],
+	'wgGalleryOptions' => [
```

Comment:
why remove the +?

Reply:
Because with the default value set, it was not needed (I don't think).

Reply:
I just added the default value so it is easier to know what to change, if it is ever requested again.